app/src/main/java/net/dogbuilt/wpi/App.java

```diff
@@ -1,11 +1,220 @@
 package net.dogbuilt.wpi;
 
+import com.github.javaparser.StaticJavaParser;
+import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
+import com.github.javaparser.ast.body.FieldDeclaration;
+import com.github.javaparser.ast.body.Parameter;
+import com.github.javaparser.ast.type.Type;
+
+import java.io.*;
```

Comment:
[Nit](https://stackoverflow.com/questions/27810522/what-does-nit-mean-in-hacker-speak): as a style rule, I always avoid wildcard imports, because they make it hard to know exactly what classes are in-scope. I suggest integrating an autoformatter like [spotless](https://github.com/diffplug/spotless), which will handle this (and most of my other nits) automatically.